frontend/projects/upgrade/src/app/features/dashboard/experiment-users/components/stratification-factor-list/stratification-factor-list.component.ts

```diff
@@ -90,6 +105,14 @@ export class StratificationComponent implements OnInit {
     });
   }
 
+  getExperimentIdsTooltip(experimentIds: any[]): string {
+    return 'Experiment IDs: [' + experimentIds.join(', ') + ']';
+  }
+
+  checkStratificationFactorUsageStatus(status: string): boolean {
+    return status === STRATIFICATION_FACTOR_STATUS.USED ? true : false;
```

Comment:
ternary isn't necessary, `status === STRATIFICATION_FACTOR_STATUS.USED` returns a boolean, leave off the `? true : false`